### DISCUSSION TITLE
工具的起源

### DISCUSSION THREAD

**Discussion by giscus** [zhuangbiaowei/zhuangbiaowei.github.io#17] [Announcements]
# 工具的起源

最近听杨照老师讲《生命的故事：达尔文与演化论60讲》，很受启发。

https://zhuangbiaowei.github.io/thinking/2023/03/28/origin-of-the-tool.html

**Comment by ZoomQuiet** [comment]
> .. 是也乎,(￣▽￣)

是暂时断言片段, 以后增补完全, 还是就这样了?

其实, 能相互改变的工具, 有个特殊属性, 好象叫 具身性?
也就是说, 可以和真实的生理结构相互交互改进的...

问题是, 人类有个特殊的生理单元 --> 大脑, 进化太超前,
应该是自然界创造出来的唯一对自然物质不满意的东西,
能自行构造出自然界不存在的各种概念和故事, 进行自我消费;

所以, 类似 AI 这种非自然产物, 可能不是传统意义上的工具,
不是谁都能拿来使用的,
是必须通过训练和教导的, 这一转化过程, 就是将自己变成一个 AI 的外部组件;

和以往工具通过连接人具身来完成增强是根本性不同的过程;

也就是说, 文化不是工具, 信息工具不是传统工具, 只有和对应有硬件交互界面结合后, 才有具身感受的信息工其, 手机就是其中形式之一, 
如果脑机接口完备可用, 
工具概念将模糊, 那是等于直接在人类意识中嵌入一个调用接口,
非常 Borg 化的根本性质性变化, 很难判定是否有好结果;
可能像 666 一样, 欢快使用多年后, 才发现不可逆转的病变,
其中, 最可能的, 就是大脑更加倾向脑机接口的信息输入,
导致一但用上, 就变成不可逆转的依赖, 比毒品还可怕,
人就完全无法离开网络接口了... etc...

嗯哼, 这种情况, 也很常见, 在各种 cyber 小说中..

**Comment by jarodtang** [comment]
这里,先有一个问题：工具是什么？我们会说，工具是人用于帮助其达成目的的辅助。这里面就引出几个问题：人是什么？从前述回答，我们知道，人是有目标述求的主体。人总想着通过做什么，达成一些目标。即如下表示
         人  ->  目标
而，有了工具，就存在如下关系
        人  - 工具  -> 目标
因此，人和工具的关系有两种看法，a）（人 - 工具）-> 目标  b) 人 -> (工具 -  目标）。中间申请各种分析过程，我们可以确定 人和工具的关系是如下的
      （人 - 工具）-> 目标
因此就有几个值得的讨论
1. 工具首先是人的拓展，不是我们用用具去做一件事，而是我们用工具拓展我们自己，而更强大的我更有能力的解决问题
2. 所以，我们通过工具来实现进化。这意味着，如何选择工具就是如何选择进化路径；已经选择与不选择就造成了区别；站在通过工具进化，这可能是一种有意识的进化路径； 这种思维也许是一种进化范式，而不仅仅是工具主义
3. 也许，我们太在乎文化的影响了，文化的基础来自于人，而工具直接改变了人，文化随着而变。比如，我们可以比较确信的说淘宝对文化影响可能远大于陈嘉应；考察一个时代的文化，从其各种工具入手可能更为可靠
